src/modules/exec.ts

```diff
@@ -31,12 +48,7 @@ const lineCapturer = (projectCache: ProjectCache, writeStdout: boolean, data: st
         if (lineTrimmed.length === 0) {
             continue;
         }
-        const result = CACHE_OUTPUT_REGEX.exec(lineTrimmed);
-        if (result && result.length > 0) {
-            const key = result[1];
-            const value = result[2].replaceAll("'", '').replaceAll('"', '');
-            projectCache.set(key, value);
-        }
+        stdOutParser(projectCache, lineTrimmed);
```

Comment:
move this whole function out of "exec.ts" to a separate module as suggested by dominic and also move the rest of the lineCapturer inside the stdOutParser so that you have it like this in the end:

```
const lineCapturer = (projectCache: ProjectCache, writeStdout: boolean, data: string) => {
    if (writeStdout) {
        process.stdout.write(data);
    }
    data.toString().split('\n').forEach(stdOutParser);
};
```